packages/auth/src/AuthProvider.tsx

```diff
@@ -16,7 +16,10 @@ export interface AuthContextInterface {
   /* Determining your current authentication state */
   loading: boolean
   isAuthenticated: boolean
-  authToken: string | null
+ /**
+  * @deprecated auth tokens are now refreshed when they expire, and this value will be removed from the context in the next few releases: https://github.com/redwoodjs/redwood/pull/1609
+  */
+  authToken: string | null // @WARN! to be depracated
```

Comment:
Small typo: `depracated` -> `deprecated`

Note: is repeated a few times, but just a single comment here.

Reply:
✅